crates/uv-python/src/implementation.rs

```diff
@@ -12,16 +12,16 @@ pub enum Error {
 
 #[derive(Debug, Eq, PartialEq, Clone, Copy, Default, PartialOrd, Ord, Hash)]
 pub enum ImplementationName {
+    GraalPy,
+    PyPy,
     #[default]
     CPython,
-    PyPy,
-    GraalPy,
 }
```

Comment:
This is pretty unintuitive, hope it doesn't bite us later :)

Reply:
I guess but it's also the standard derive implementation!